windows/vhba_hot_add_remove/set_boot_disk.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Set the boot disk to the specified one.
+# Parameters:
+#   hdd_name: the specified boot disk name. E.g. 'Hard disk 1' which can be got from vSphere Web UI
+#
+- name: Check required parameter
+  ansible.builtin.assert:
+    that:
+      - hdd_name is defined
+      - hdd_name
+    fail_msg: "hdd_name is not defined"
+
+# Shutdown guest OS
+#- include_tasks: ../utils/win_shutdown_restart.yml
+#  vars:
+#    set_win_power_state: "shutdown"
+
+# Set the boot disk to the specified disk
+- include_tasks: ../../common/vm_set_boot_options.yml
+  vars:
+    boot_hdd_name: "{{ hdd_name }}"
+    boot_order_list:
+      - disk
+
```

Comment:
If there is no more includes of `set_boot_disk.yml`, I think this file can be removed, and you can directly put the `include_tasks` into `windows/vhba_hot_add_remove/create_new_file_read_write.yml`

Reply:
Waiting for all the testing results. If no power operations are needed indeed. Will restruct the code.